tests/openQCD_in_test.py

```diff
@@ -7,7 +7,7 @@
 def test_rwms():
     path = './tests//data/openqcd_test/'
     prefix = 'sfqcd'
-    postfix = '.rwms'
```

Comment:
Does this PR change the behavior of the input functions? Would using `.rwms` still work?

Reply:
Originally, it did, however after I realized this would be a bit unwieldy and it would be pretty easy to correct, I corrected it here: https://github.com/fjosw/pyerrors/pull/151/commits/fbf13493624023014c4d279bc391c9136ebe8bbf. Like this, everything works, ```rwms```, ```.rwms```, ```rwms.``` (whoever would use this) and ```.rwms.```

Reply:
Very good, I think the test should then cover a few of these cases.

Reply:
Sure, why not... however I would suggest to put this into the `test_find_files`, as that makes it simpler for later bug-fixing.